src/main/java/edu/kis/powp/jobs2d/shapes/RectangleShape.java

```diff
@@ -0,0 +1,107 @@
+package edu.kis.powp.jobs2d.shapes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+/**
+ * This class represents a rectangular shape. It offers two ways to create:
+ * 1) by explicit width and height,
+ * 2) by a known format name (e.g., "a4", "b3").
+ */
+
+public class RectangleShape implements Shape{
+    private final int width;
+    private final int height;
+
+    /**
+     * Create a RectangleShape with given dimensions.
+     * 
+     * @param width width of the rectangle
+     * @param height height of the rectangle
+     */
+    public RectangleShape(int width, int height) {
+        this.width = width;
+        this.height = height;
+    }
+	
+    /**
+     * Create a RectangleShape with given format.
+     * 
+     * @param format format of the rectangle
+     * @throws IllegalArgumentException if the format is not supported.
+     */
+    public RectangleShape(String format) {
+    	switch(format.toLowerCase()) {
+    	    case "a4":
+    	        this.height = 210;
+    	        this.width = 297;
+    	        break;
+    	    case "a3":
```

Comment:
The identifiers ("a4", etc.) should have single source of truth (SSOT), e.g., be replaced with enum.
I would prefer to convert this method to factory methods, rather to have such obscure constructor.